### DISCUSSION TITLE
Paper trail

### DISCUSSION THREAD

**Discussion by npearson72** [jotaijs/jotai-devtools#113] [Ideas]
I have cases where tracking the "initial" state (of data that changes over time)  is important. I know I can create a separate atom to capture that, but am wondering if this is a common enough use-case where it might be worth considering a paper trail feature?

**Comment by arjunvegda** [comment]
Hey @npearson72 

We have a time travel feature (Demo [here](https://k5p12d.csb.app/)) that lets you track _all_ your atoms at once. That should solve your usecase. What do you think?  😁


<img width="1519" alt="image" src="https://github.com/jotaijs/jotai-devtools/assets/14841132/4a47d0a8-fbd6-443b-9033-f43d57a615e0">

**Reply by npearson72** [reply]
I actually need this in my runtime code, not just as a development feature. Is there a way to access this as general state in my code?

**Reply by arjunvegda** [reply]
Unfortunately not, Jotai Devtools and its dev/debug-related methods are only available when `process.env.NODE_ENV !== 'production'`
